cli/Cargo.toml

```diff
@@ -63,4 +63,5 @@ tokio = { version = "1.6.0", features = ["sync", "time", "rt", "io-util", "rt-mu
 warp = "0.3"
 
 [dev-dependencies]
+serial_test = "0.8.0"
```

Comment:
Using nextest as standard is still the preferred solution. But this guarantees that the tests pass locally using just `cargo test`.